sensors:
  right_bins_camera:
    type: rgbd_camera
    pose:
      xyz: [-2.286, 2.96, 1.8]
      rpy: [pi, pi/2, 0]

  left_bins_camera:
    type: rgbd_camera
    pose:
      xyz: [-2.286, -2.96, 1.8]
      rpy: [pi, pi/2, 0]

  kts1_camera:
    type: rgbd_camera
    pose:
      xyz: [-1.3, -5.8, 1.8]
      rpy: [pi, pi/2, pi/2]
  
  kts2_camera:
    type: rgbd_camera
    pose:
      xyz: [-1.3, 5.8, 1.5]
      rpy: [pi, pi/2, -pi/2]
  
  
  